lte/gateway/c/session_manager/test/test_local_enforcer.cpp

```diff
@@ -55,15 +49,25 @@ class LocalEnforcerTest : public ::testing::Test {
     auto default_mconfig = get_default_mconfig();
     local_enforcer       = std::make_unique<LocalEnforcer>(
         reporter, rule_store, *session_store, pipelined_client,
-        directoryd_client, events_reporter, spgw_client,
-        aaa_client, 0, 0, default_mconfig);
+        directoryd_client, events_reporter, spgw_client, aaa_client, 0, 0,
+        default_mconfig);
     evb = folly::EventBaseManager::get()->getEventBase();
     local_enforcer->attachEventBase(evb);
     session_map = SessionMap{};
+    initialize_lte_test_config();
   }
 
   virtual void TearDown() { folly::EventBaseManager::get()->clearEventBase(); }
 
+  void initialize_lte_test_config() {
+    test_cfg_.spgw_ipv4 = "128.0.0.1";
+    build_common_context(
+        "", "127.0.0.1", "IMS", "", TGPP_LTE, &test_cfg_.common_context);
+    build_lte_context(
+        "128.0.0.1", "", "", "",
+        test_cfg_.rat_specific_context.mutable_lte_context());
+  }
+
```

Comment:
In the future, if we keep doing refactors, it would be good to look for all those functions that use this pattern and revert it. 

It is so easy to read the code when there is a proper return 😍
```
  void initialize_lte_test_config() {
    test_cfg_.spgw_ipv4 = "128.0.0.1";
    test_cfg_.common_context = build_common_context(
        "", "127.0.0.1", "IMS", "", TGPP_LTE);
    test_cfg_.rat_specific_context.mutable_lte_context = build_lte_context(
        "128.0.0.1", "", "", "");
  }

```

Reply:
Will modify these back in the top most diff :) (rebasing stacked PRs is a bit of a hassle)